src/storage/mod.rs

```diff
@@ -770,6 +779,9 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
                     let begin_instant = Instant::now_coarse();
                     let mut stats = Statistics::default();
                     let r = Self::raw_get_key_value(&snapshot, cf, key, &mut stats);
+
+                    let _g = minitrace::new_span(tracing::Event::TiKvCollectMetrics as u32);
```

Comment:
how about:

```rust
{
   let _g = ...
}
```

to exclude Drop / send time of `r` and other things

Reply:
I don't really want to trace `TiKvCollectMetrics`. It's also for demonstration and will be removed if the rest is ok.

Reply:
Got it